jedi/parser/tree.py

```diff
@@ -317,7 +317,7 @@ def nodes_to_execute(self, last_added=False):
 
     @utf8_repr
     def __repr__(self):
-        return "<%s: %s>" % (type(self).__name__, self.value)
+        return "<%s: %s %s-%s>" % (type(self).__name__, self.value, self.start_pos, self.end_pos)
```

Comment:
Hmm, I would prefer if this was just the start position. It's harder for me when debugging, otherwise (When enabling Jedi's internal debugging).